app/screens/theme/theme.test.js

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FlatList} from 'react-native';
+import {shallow} from 'enzyme';
+import Section from 'app/screens/settings/section';
+
+import Preferences from 'mattermost-redux/constants/preferences';
+
+import Theme from './theme';
+
+jest.mock('react-intl');
+
+describe('Theme', () => {
+    const baseProps = {
+        teamId: 'test-team',
+        theme: Preferences.THEMES.default,
+        userId: 'test-user',
+        actions: {
+            savePreferences: jest.fn(),
+        },
+        allowedThemes,
+    };
+
+    test('should match snapshot', () => {
+        const wrapper = shallow(
+            <Theme {...baseProps}/>,
+        );
+
+        expect(wrapper.getElement()).toMatchSnapshot();
+        expect(wrapper.find(Section).first().exists()).toEqual(true);
+        expect(wrapper.find(FlatList).first().exists()).toEqual(true);
+    });
+});
+
+const allowedThemes = [
```

Comment:
You might be able to some space by just using `Preferences.THEMES` here, although thinking more about it, that would also make this test break if mattermost-redux changes the themes so that may not be the best approach. Up to you if you'd want to change that